apps/site/pages/en/learn/typescript/run-natively.md

```diff
@@ -36,6 +55,10 @@ At the time of writing, the experimental support for TypeScript in Node.js has s
 
 You can get more information on the [API docs](https://nodejs.org/docs/latest/api/typescript.html#typescript-features).
 
+### Configuration
+
+Node.js will not support `tsconfig.json` by default. This means that, for a seamless experience while using TypeScript with Node, a base `tsconfig.json` configuration is required in order to match what Node achieves using [Amaro](https://github.com/nodejs/amaro) (Node's TS loader). Such configuration can be found [here](https://nodejs.org/api/typescript.html#type-stripping) using TypeScript on version **5.7 or higher**.
```

Comment:
This section initially confused me; I think I'd find this clearer if it said something like:

> Node.js's TypeScript loader (Amaro) does not consult `tsconfig.json` when transforming TypeScript code. In order to configure TypeScript to recognize this behavior when type checking, we recommend setting the `compilerOptions` listed in <link here>.

Reply:
That's true, updating it here